example/micro2

```diff
@@ -0,0 +1,43 @@
+#!/bin/zsh
```

Comment:
What's this script?

Reply:
Just a script for testing, must've done a `git add .` or `git add <directory>` by accident. Both files should be gone now.